torch/nn/modules/pooling.py

```diff
@@ -685,6 +694,16 @@ class AvgPool3d(_AvgPoolNd):
               W_{out} = \left\lfloor\frac{W_{in} + 2 \times \text{padding}[2] -
                     \text{kernel\_size}[2]}{\text{stride}[2]} + 1\right\rfloor
 
+          If ``ceil_mode`` is True and :math:`(D_{out} - 1)\times \text{stride}[0]\geq D_{in}
+          + \text{padding}[0]`, we skip the last window, resulting in :math:`D_{out}` being
+          reduced by one.
+
+          Same for :math:`H_{out}`, if ``ceil_mode`` is True and :math:`(H_{out} - 1)\times
+          \text{stride}[1]\geq H_{in} + \text{padding}[1]`, :math:`H_{out}` is reduced by one.
+
+          And for :math:`W_{out}`, if ``ceil_mode`` is True and :math:`(W_{out} - 1)\times
+          \text{stride}[2]\geq W_{in} + \text{padding}[2]`, :math:`W_{out}` is reduced by one.
```

Comment:
nit: Similar to above 

"The same applies for :math:`W_{out}` and :math:`H_{out}`"

Reply:
Done! Thanks